components/Footer.js

```diff
@@ -0,0 +1,206 @@
+import NavLink from './NavLink'
+
+/**
+ * Footer for the main layout.
+ * Derived from https://codepen.io/tutsplus/pen/yWrEgW
+ */
+export default function Footer (props) {
+  return (
+    <footer>
+      <section className='ft-main'>
+        <div className='ft-main-item'>
+          <h2 className='ft-title'>About</h2>
+          <ul>
+            <li>
+              <NavLink
+                // FIXME: Update content at link
+                href='/about'>
+                About this app
+              </NavLink>
+            </li>
+            <li>
+              <NavLink
+                // FIXME: Update content at link
+                href='/faq'>
+                FAQ
+              </NavLink>
+            </li>
+            <li>
+              <NavLink href='https://fdot-rmce.statuspage.io/' external>
+                Status
+              </NavLink>
+            </li>
+          </ul>
+        </div>
+        <div className='ft-main-item'>
+          <h2 className='ft-title'>Resources</h2>
+          <ul>
+            <li>
+              <NavLink
+                // FIXME: Update link
+                href='https://fdot-otp.ibi-transit.com/'
+                external
+              >
+                View Trip Planner
+              </NavLink>
+            </li>
+            <li>
+              <NavLink
+                // FIXME: Update link
+                href='#'
+                external
+              >
+                API documentation
+              </NavLink>
+            </li>
+          </ul>
+        </div>
+        <div className='ft-main-item'>
+          <h2 className='ft-title'>Contact</h2>
+          <ul>
+            <li>
+              <NavLink
+                // FIXME: Update link
+                href='/'>
+                Help
+              </NavLink>
+            </li>
+            <li>
+              <NavLink
+                // FIXME: Update link
+                href='/'>
+                Request access to API
```

Comment:
I am torn on this one. It fits with the Help heading, but is not relevant to the admin dashboard user. Also, it duplicates the API sign up link in the nav bar.

Reply:
I'm going to keep this in place for now. It's not uncommon for navbars and footers to have redundant content. The goal here is to help people navigate the website, not necessarily to be minimal in expression/content.